modules/minimp3/audio_stream_mp3.h

```diff
@@ -47,6 +47,8 @@ class AudioStreamPlaybackMP3 : public AudioStreamPlaybackResampled {
 	AudioFrame loop_fade[FADE_SIZE];
 	int loop_fade_remaining = FADE_SIZE;
 
+	bool looping_override = false;
+	bool looping = false;
```

Comment:
This could be an enum. `looping_override = false` and `looping = true` is invalid state.
Though it's fine like this too.